CHANGELOG.md

```diff
@@ -26,6 +26,25 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   structure by separating concerns, as these traits are not directly related to
   the GraphQL API but are instead utilized within it.
 - Update `TestSchema` to accept `test_addr` for simulating client IP.
+- Bump dependencies.
+  - Update async-graphql to version "7".
+  - Update async-graphql-axum to version "7".
+  - Update axum to version "0.7".
+  - Update axum-server to version "0.6".
+  - Update http to version "1.1".
+  - Update reqwest to version "0.12".
+  - Update review-protocol to vesion "0.3.0"
+  - Update rustls to version "0.22".
+  - Update rustls-native-certs to version "0.7"
+  - Update rustls-pemfile to version "2.0".
+  - Update tower-http to version "0.5"
```

Comment:
I don't think a simple list of bumping conveys any useful meaning. Only significant or noticeable changes should be mentioned for brevity and clarity.

Reply:
done